diagnostic.md

```diff
@@ -5,27 +5,30 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    A list of quotes and within the list are different famous people who have quotes so say winston churchill's quotes are a component within the quotes component.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+ember generate component my-map
     ```
 
 1.  What files are created and/or edited to produce a component, and what are their responsibilities?
 
     ```md
-    <!-- your response here -->
+    I tried looking this up but couldn't find it. I think its the component file & template file are created, & the route is updated.
+    The component tells what actions to do and or show. the template shows what will be on the front end. the route is updated in order
+    to show where the data will be displayed
```

Comment:
The Router is updated when we generate a Route, but not when we generate a component. 

But other than that this answer is correct!
1/1